source/production/F0.Minesweeper.App/Pages/Index.razor

```diff
@@ -1,8 +1,4 @@
 @page "/"
 
-<h1>Hello, world!</h1>
-
-Welcome to Minesweeper, powered by Blazor.
-
-
-<Minefield Options="@MinefieldOptions" />
+@*Index is only used for automatic redirect to the game component*@
+<div />
```

Comment:
Stupid question, but do we need this `div` or can we omit it? :)

Reply:
Not really a stupid question tbh. I had no clue whether we can so I just tested the game without it and there seem to be no repercussions.
Great find!